client/app/styles/_before_after_image_slider.scss

```diff
@@ -4,8 +4,38 @@
   & > * {
     position: absolute;
   }
-  .second-wrapper {
+
+  .labels {
+    height: 25px;
+    position: relative;
+    font-family: Menlo, Monaco, Consolas, "Courier New", monospace;
+    margin-left: 2px;
+    margin-right: -2px;
+
+    .baseline {
+      float: left;
+    }
+
+    .regression {
+      float: right;
+    }
+  }
+
+  .baseline-wrapper {
+    height: 100%;
+    border-left: 2px solid green;
+    border-top: 2px solid green;
+    border-bottom: 2px solid green;
+    border-right: 1px solid black;
+    background-color: transparent;
```

Comment:
It's fix for half display. If you do not want transparent we can change it to any another color or picture.